static/src/javascripts/projects/common/modules/identity/upsell/account-creation/AccountBenefits.js

```diff
@@ -0,0 +1,111 @@
+// @flow
+import React from 'preact-compat';
+import config from 'lib/config';
+
+type BenefitCta = {
+    name: string,
+    link: string,
+};
+
+type Benefit = {
+    title: string,
+    blurb: string,
+};
+
+type ActionableBenefit = Benefit & {
+    cta: BenefitCta[],
+};
+
+const benefits: Benefit[] = [
+    {
+        title: `Get closer to our journalism`,
```

Comment:
you can use single quotes for all of these, there's no interpolation.

Reply:
These strings can potentially contain single quote apostrophes which would require to switch them to backticks, so we might as well use these to err on the safe side as there's no performance hit (babel will compile them into single quotes)